recipes/get_version/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "get_version" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://github.com/flying-sheep/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ed320df3b0604ecab925f5e00290c09f85c169bc8f826bb263b0d82889be487c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m flit install --deps none"
```

Why ``flit``? should it be :
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv
```

---

Why not flit? This is one more indirection and flit already does everything.

---

and using `pip` it fails on windows due to a `UnicodeDecodeError`. 
See [appveyor build](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/27555974)

---

and using `pip` it fails on windows due to a `UnicodeDecodeError`. 
See [appveyor build](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/27555974)